src/hooks/use_react_window.ts

```diff
@@ -8,8 +8,8 @@ import * as R from 'ramda';
 // reusable hook helpers for react window list components
 
 export const useList = () => {
-  const listRef = useRef({});
-  const rowHeights = useRef({});
+  const listRef:any = useRef();
```

Comment:
I'm not sure if this breaks anything but we should follow the rule, `If it's not broken. Don't fix it`.

Reply:
I fix because it was broken~